Bookish.Web/Controllers/HomeController.cs

```diff
@@ -24,10 +25,17 @@ public IActionResult Catalogue(string filter = "")
             return View(model);
         }
 
-        public IActionResult Copies(string isbn, string title, string author)
+        [Route("/Home/Copies/{isbn}")]
+        public IActionResult Copies(string isbn)
         {
-            var copies = bookishService.GetCopies(isbn);
-            var model = new CopiesViewModel(title, author, copies);
+            var copies = bookishService.GetCopies(isbn).ToList();
+
+            if (!copies.Any())
+            {
+                return RedirectToAction("Error", "Home");
```

Comment:
You don't need to do this as part of this PR but I think this should redirect to a 404 page with a suitable error.  I'm not sure what status code your error function is currently using, probably worth checking when you come to deal with error handling properly, as well as checking that you're happy with the error message being displayed 🙂